src/SmartComponents/Inventory/applications/compliance/compliance.scss

```diff
@@ -2,7 +2,20 @@
     text-align: center;
 }
 
-#remediation-button > button {
-    padding-left: 0.625rem;
-    padding-right: 0.625rem;
+.ins-c-compliant-card {
+    font-size: large;
```

Comment:
We really shouldn't use absolute font sizes - `small, medium, large, etc.` . I'd say to establish it with the [rem converter](https://github.com/RedHatInsights/insights-frontend-components/blob/master/src/Utilities/_mixins.scss).